src/main/drivers/pwm_output_stm32f4xx.c

```diff
@@ -167,14 +167,14 @@ void pwmDigitalMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t
 
     TIM_OCStructInit(&TIM_OCInitStructure);
     TIM_OCInitStructure.TIM_OCMode = TIM_OCMode_PWM1;
-    if (timerHardware->output & TIMER_OUTPUT_N_CHANNEL) {
+    if (output & TIMER_OUTPUT_N_CHANNEL) {
```

Comment:
Layers are mixed here - TIMER_OUTPUT_N_CHANNEL is timer channel specific (pin mapping depends on it), output polarity is user specified - ideally they shall be separate. In this case passing inversion as parameter independent on `timerHardware->output` will help, but there will be similar cases elsewhere when it won't work well.